pom.xml

```diff
@@ -25,29 +25,23 @@
 
   <developers>
     <developer>
-      <name>Chris Jordan</name>
```

Comment:
I think you can leave Chris on here. If anyone actually sent mail to `chris@messageyes.com` we'll see it in the undeliverable queue.

Reply:
I added him back.